src/lib/zAuction.ts

```diff
@@ -87,8 +103,41 @@ export async function getBidsForNft(
 	return bids;
 }
 
+export async function getBidsListForNfts(
+	baseApiUri: string,
+	contract: string,
+	tokenId: string[],
+) {
+	const nftIdsArray: string[] = [];
```

Comment:
Putting `Array` in variable names is redundant in Typescript since you can imply that something is an array based on it it's type. Having a name like `nftIds` would be wiser here since the user can easily see that it is a `string[]`